src/kinova_gen3_mujoco_config/README.md

```diff
@@ -0,0 +1,5 @@
+# picknik_panda_mujoco_config
+
+A MoveIt Pro configuration to simulate a Panda Franka robot using mock hardware.
```

Comment:
nit: replace panda with kinova, and mock hardware with mujoco (this can probably be done at the end once everything else in this PR is ready in case we have other things we want to add to the README)

Reply:
Done